spec/index.bs

```diff
@@ -1888,6 +1889,8 @@ interface GPUAdapterInfo {
     readonly attribute DOMString architecture;
     readonly attribute DOMString device;
     readonly attribute DOMString description;
+    readonly attribute unsigned long minSubgroupSize;
+    readonly attribute unsigned long maxSubgroupSize;
```

Comment:
I personally don't find it problematic, but I'm open to suggestions. The bounds don't seem that different to me. What about `minSupportedSubgroupSize` and `maxSupportedSubgroupSize`?

Reply:
I think `maxSupportedSubgroupSize` still doesn't solve the problem.  It still sounds like the programmer tries something and the system accepts or denies.  Also "supported" is used in the name of `GPUSupportedLimits` to bound the things the programmer can do.

Second, in future the causality will be flipped around.
We're likely to add a future feature to let the programmer control the subgroup size. There are good reasons to do so, it's been requested (#3950), and more recent D3D and Vulkan can support it.  It's not in this round because Metal can't. See the TODO at https://github.com/gpuweb/gpuweb/blob/main/proposals/subgroups.md#gpu-feature 

I think the current names are good.  (Also, it helps that Vulkan uses the same names. No reason to make a new name for the same concep.t)

Reply:
The Vulkan feature is accessed at pipeline creation and the D3D feature is a shader attribute.

Vulkan has 2 pipeline creation flags:
* one allows varying subgroup sizes
* one requires full pipelines

Vulkan also has a pipeline creation structure to specify a specific subgroup size. It is not compatible with the varying subgroup size flag, but the full pipeline bit is optional.

D3D can specify a required subgroup size via an attribute in SM6.6.

The usual case is that the shader author has specialized versions of the shader for specific subgroup sizes.

So this additional functionality isn't really limits. It would be more like a shader/pipeline value to specify a specific subgroup size. That value could be validated against the adapter info being added here (otherwise you'd expect failed compiles).

Reply:
`subgroupMinSize` and `subgroupMaxSize` (matching the proposal) SGTM. I'll make that change.